pages/my-library/index.js

```diff
@@ -89,6 +89,8 @@ export default function MyLibraryPage({
 
 const StyledHeadline = styled.h2`
   text-align: center;
+  margin: auto;
+  margin-bottom: 20px;
```

Comment:
Shorthand for top, right and left (auto), and bottom (20px) margins.

```suggestion
margin: auto auto 20px;
```